### PR TITLE
Wrapped label encoder

### PR DESCRIPTION
**Pull request by kxk302**
This is to encode categorical class labels (into unique integers).

### CODE REVIEW

**Comment by kxk302** [comment]
Made minor revisions after a self review. This is ready for review now. Thanks!

File: tools/sklearn/label_encoder.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <param name="header0" value="false"/>
+            <output name="outfile" file="le_output.tabular" ftype="tabular"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+**What it does**
+
+class sklearn.preprocessing.LabelEncoder
+
+Encode target labels with value between 0 and n_classes-1.
```
[the remark this answers is not in the document]
Comment (kxk302) [reply tools/sklearn/label_encoder.xml:43]: Revised help accordingly.

**Comment by bgruening** [comment]
Merci!
